.github/workflows/helpers/pull-request-utils.js

```diff
@@ -409,31 +409,8 @@ class coverageHelper {
   }
 }
 
-class userHelper {
-  constructor(input) {
-    this.owner = input.context.repo.owner
-    this.repo = input.context.repo.repo
-    this.github = input.github
-  }
-
-  /*
-    Checks if the user has write permissions for the repository
-    @returns {boolean} - returns true if the user has write permissions, otherwise false
-  */
-  async hasWritePermissions() {
-    const { data } = await this.github.rest.repos.getCollaboratorPermissionLevel({
-      owner: this.owner,
-      repo: this.repo,
-      username: this.owner,
-    })
-    console.log(JSON.stringify(data))
-    return data.permission === writePermission || data.permission === adminPermission
-  }
```

Comment:
Just for my understanding - what was the problem with this script?

Reply:
release workflow was using `actions/github-script` to check if user has write permission. However actions/github-script was not returing user permission. Refer https://github.com/prebid/prebid-server/actions/runs/7912092232/job/21597265232 for more details.

<img width="1727" alt="Screenshot 2024-02-15 at 1 33 54 PM" src="https://github.com/prebid/prebid-server/assets/24757781/2a269951-d790-4e3e-8827-173de3e80637">